### DISCUSSION TITLE
Multiple name_server with DHCP?

### DISCUSSION THREAD

**Discussion by tompaah** [BeryJu/gravity#1334] [Q&A] [answered]
I'd like to pass all my three Gravity instances as DNS servers in DHCP offers. Tried the following but none works.
Client only gets the last entry.
```
- tagName: name_server
  value: 192.168.1.2
- tagName: name_server
  value: 192.168.1.3
- tagName: name_server
  value: 192.168.1.4

```
Client doesn't interpret this correctly
```
- tagName: name_server
  value: 192.168.1.2,192.168.1.3,192.168.1.4

```
Gravity UI doesn't accept this as syntax
```
- tagName: name_server
  value:
    - 192.168.1.2
    - 192.168.1.3
    - 192.168.1.4
```

Is there a supported way of providing multiple DNS servers to DHCP clients?

**Comment by ctjutting** [comment]
Did you find a way to make this work? I'm looking into the same problem right now and not seeing the proper syntax.

**Reply by tompaah** [reply]
Sorry, no.
Had to pause my deployment of Gravity due to issue [https://github.com/BeryJu/gravity/issues/1350](url). Hoping to reinstate it in the future, it is awesome by many means.

**Reply by ctjutting** [reply, marked as answer]
FYI, This works. Move to the tag not the tagName and convert each IP to hex with whatever converter works online. I have all 4 of my nodes now working like I want it too. I cut over this weekend

- tag: 6
  tagName: ""
  valueHex:
    - 0a32fcdc
    - 0a32fce1
    - 0a32fce6
    - 0a3265f0
